engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/KafkaAvroSourceFactorySpec.scala

```diff
@@ -54,18 +56,18 @@ class KafkaAvroSourceFactorySpec extends FunSpec with BeforeAndAfterAll with Kaf
     kafkaClient.createTopic(IntTopic, 2)
   }
 
-  it("should read last generated record") {
+  it("should read generated record in v1") {
```

Comment:
Let's change it to FunSuite, Idea still has problems with FunSpec...

Reply:
Changed